recipes/pti-gpu-unitrace/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pti-gpu-unitrace" %}
+# TODO: set version after original package is tagged
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # TODO: set version after original package is tagged
+  git_url: https://github.com/intel/pti-gpu.git
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  ignore_run_exports:
+    - dpcpp-cpp-rt
+
+requirements:
+  build:
+    - cmake >=3.29
+    - ninja >=1.11.1
+    - {{ compiler('cxx') }}
+    - {{ compiler('dpcpp') }} >=2024.1.0
+    - {{ stdlib('c') }}
+  host:
+    - level-zero-devel >=1.16
+    - {{ pin_compatible('dpcpp-cpp-rt', min_pin='x.x', max_pin='x') }}
+    - {{ pin_compatible('intel-cmplr-lib-rt', min_pin='x.x', max_pin='x') }}
+  run:
+    - {{ pin_compatible('intel-cmplr-lib-rt', min_pin='x.x', max_pin='x') }}
+    - {{ pin_compatible('ocl-icd', min_pin='x.x', max_pin='x') }}
+    - {{ pin_compatible('level-zero', min_pin='x.x', max_pin='x') }}
```

Comment:
All of these probably should have a run_exports and thus should not be listed explicitly here.

Reply:
Unfortunetely no.

Without those pins I have warnings during the build time:
```
Packaging pti-gpu-unitrace
Packaging pti-gpu-unitrace-0.0.1-h07fc403_0
number of files: 13
   INFO: sysroot: '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone1970.tab', '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone.tab', '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/tzdata.zi', '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/Zulu']'
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
WARNING (pti-gpu-unitrace,bin/unitrace): Needed DSO lib/libze_loader.so.1 found in ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0']
WARNING (pti-gpu-unitrace,bin/unitrace): .. but ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO lib/libstdc++.so.6 found in conda-forge/linux-64::libstdcxx-ng==13.2.0=hc0a3c3a_7
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-64::libgcc-ng==13.2.0=h77fa898_7
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): $RPATH/libxptifw.so not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): $RPATH/libOpenCL.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO lib/libze_loader.so.1 found in ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0']
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): .. but ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO lib/libstdc++.so.6 found in conda-forge/linux-64::libstdcxx-ng==13.2.0=hc0a3c3a_7
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-64::libgcc-ng==13.2.0=h77fa898_7
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): $RPATH/libintlc.so.5 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
Fixing permissions
Packaged license file/s.
INFO :: Time taken to mark (prefix)
        0 replacements in 0 files was 0.02 seconds
fatal: No names found, cannot describe anything.
WARNING: Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
```

And they are not added to the released package:

```
Loading channels: done
pti-gpu-unitrace 0.0.1 h07fc403_0
---------------------------------
file name   : pti-gpu-unitrace-0.0.1-h07fc403_0.conda
name        : pti-gpu-unitrace
version     : 0.0.1
build       : h07fc403_0
build number: 0
size        : 554 KB
license     : MIT
subdir      : linux-64
url         : file:///home/yevhenii/.miniforge3/conda-bld/linux-64/pti-gpu-unitrace-0.0.1-h07fc403_0.conda
md5         : 5cf84c599f7682c8330470ce9be5c506
timestamp   : 2024-05-20 17:34:16 UTC
dependencies:
  - __glibc >=2.17,<3.0.a0
  - libgcc-ng >=13
  - libstdcxx-ng >=13
```

Reply:
Then please make a PR or file an issue with the feedstock that they are missing run_exports and reference it here.

Reply:
I've fixed `level-zero-feedstock` and opened an issues on `intel-compilers-repack-feedstock`.